docs/semantic-conventions.md

```diff
@@ -23,12 +23,17 @@ are implemented by Java autoinstrumentation and which ones are not.
 | `http.route` | N | - |
 | `http.client_ip` | N | + |
 
-**[1]:** As the majority of Java frameworks don't provide a standard way to obtain "The full request
+**[1]:** Most server instrumentation captures `http.scheme`, `http.host`, and `http.target`
+(and does not capture `http.url`).
+Netty instrumentation is currently the only exception to this rule. Netty instrumentation
+captures `http.url` (and does not capture `http.scheme`, `http.host`, or `http.target`).
+
+As the majority of Java frameworks don't provide a standard way to obtain "The full request
```

Comment:
This paragraph and the previous one are in direct contradiction to one another

Reply:
WDYT about rewriting that paragraph (splitting in two?) so that it describes servers and clients separately?

Reply:
> This paragraph and the previous one are in direct contradiction to one another

oops I should have deleted this paragraph, it's gone now

> WDYT about rewriting that paragraph (splitting in two?) so that it describes servers and clients separately?

http client is split out below